factories/esSearcherFactory.js

```diff
@@ -149,6 +149,41 @@
         }
       };
 
+      var formatError = function(msg) {
+          var errorMsg = '';
+          if (msg) {
+            if (msg.status >= 400) {
+              errorMsg = 'HTTP Error: ' + msg.status + ' ' + msg.statusText;
+            }
+            if (msg.status > 0) {
+              if (msg.hasOwnProperty('data') && msg.data) {
+
+                if (msg.data.hasOwnProperty('error')) {
+                  errorMsg += '\n' + JSON.stringify(msg.data.error, null, 2);
+                }
+                if (msg.data.hasOwnProperty('_shards')) {
+                  angular.forEach(msg.data._shards.failures, function(failure) {
+                    errorMsg += '\n' + JSON.stringify(failure, null, 2);
+                  });
+                }
+
+              }
+            }
+            else if (msg.status === -1) {
+              errorMsg +=  'Network Error! (host not found)\n';
+              errorMsg += '\n';
+              errorMsg +=  'or CORS needs to be configured for your Elasticsearch\n';
+              errorMsg +=  '\n';
+              errorMsg +=  'Enable CORS in elasticsearch.yml:\n';
+              errorMsg += '\n';
+              errorMsg += 'http.cors.allow-origin: "/.*/"\n';
```

Comment:
@softwaredoug should we make this more specific to Quepid? 

`/https?:\/\/.+\.quepid\.com/`